site/src/component/SideBar/SideBar.tsx

```diff
@@ -74,14 +70,19 @@ const SideBar = ({ darkMode, toggleTheme }: SideBarProps) => {
         </NavLink>
       </li>
       {showSidebar && <li>
-        <a style={{cursor: 'pointer'}} onClick={toggleTheme}>
-          <div>
-            <Icon name={darkMode ? 'sun outline' : 'moon outline'} size='large' />
-          </div>
-          <span>
-            {darkMode ? 'Toggle Light Theme' : 'Toggle Dark Theme'}
-          </span>
-        </a>
+        <ThemeContext.Consumer>
+          {({ darkMode, toggleTheme }) => 
+            // eslint-disable-next-line jsx-a11y/anchor-is-valid
+            <a className='theme-toggle' onClick={toggleTheme}>
```

Comment:
why are we using an `a` and onclick as opposed to a different element? If it's just for styling we should probably use a different element

Reply:
I think I was just too lazy too change the styling for a different element at the time. Should probably change it to a button but might just leave it for now. This is a temporarily a place for the toggle. In the future, I would like the redesign the header and nav menu and have a profile menu in the top right.

![image](https://github.com/icssc/peterportal-client/assets/8922227/4b3a15d9-adaa-447b-8d1d-2338688d29ff)

Idea is that the theme button would expand to have system, light, and dark theme options, kind of like YouTube.

![image](https://github.com/icssc/peterportal-client/assets/8922227/172dbdd2-6739-4074-b139-59130d264b35)
![image](https://github.com/icssc/peterportal-client/assets/8922227/c94969fb-e33e-4f7e-990e-de387b9fdf03)

And yes, jsx-a11y was removed so the eslint-disable was not needed. We might consider adding it back in the future, but it was too much trouble at the time to incorporate in the vite PR due to the abundance of accessibility errors we have.

Reply:
I see. I think if there's not any warnings then we can leave it for now until we make changes to the sidebar and/or user preferences